.github/workflows/run-api-analytics-tests.yml

```diff
@@ -0,0 +1,79 @@
+name: Run analytics api tests
+env:
+  # This is to make sure Maven don't timeout fetching dependencies. See: https://github.com/actions/virtual-environments/issues/1499
+  MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=125
+
+on:
+  pull_request:
+    types: [ opened, labeled, synchronize ]
+  schedule:
+    - cron: "16 1 * * *" # Run at 01:16 AM
```

Comment:
It only runs on labeled PRs and not on master. @maikelarabori can elaborate on this

Reply:
From what I understood from our conversation @teleivo, it should run only on labelled PRs and once a day in the master branch.
If you prefer to run only on labelled PRs in this first stage, I'm good with that.

Reply:
Good point @radnov !
@maikelarabori I added a slack notification to this workflow. Should we send it to the analytics team instead of the team-backend channel? If so please adjust `SLACK_CHANNEL`

Reply:
Looks good @teleivo